src/dns/server.rs

```diff
@@ -1627,23 +1656,31 @@ mod tests {
                 "headless.pod0.ns1.svc.cluster.local",
                 &NW1,
                 &[format!("{}/{}", NS1, kube_fqdn("headless", NS1)).as_str()],
-                &[ip("30.30.30.30")],
+                &[ip("30.30.30.30"), ip("2001:db8::30")],
```

Comment:
For my own context, where do these IPs come from?

Reply:
the 30.30.30.30 is just random garbage though